### DISCUSSION TITLE
Error: Chrome did not write a file at `screenshot.png`

### DISCUSSION THREAD

**Discussion by sajedulsakib001** [spatie/browsershot#602] [Q&A]
I am getting an error. How can i fix it.
I am using cpanel hosting 
Thanks

`
Fatal error: Uncaught Spatie\Browsershot\Exceptions\CouldNotTakeBrowsershot: For some reason Chrome did not write a file at "screenshot.png". Command payload: [] in /home/username/vendor/spatie/browsershot/src/Exceptions/CouldNotTakeBrowsershot.php:13 Stack trace: #0 /home/username/vendor/spatie/browsershot/src/Browsershot.php(601): Spatie\Browsershot\Exceptions\CouldNotTakeBrowsershot::chromeOutputEmpty('screenshot.png') #1 /home/username/public_html/a.php(8): Spatie\Browsershot\Browsershot->savePdf('screenshot.png') #2 {main} thrown in /home/username/vendor/spatie/browsershot/src/Exceptions/CouldNotTakeBrowsershot.php on line 13

`

**Comment by IslamulHadi** [comment]
did you fix it? I got same error.

**Reply by go4nishu** [reply]
I am also getting same error. anybody found the solution of this error?

**Comment by sbc-mining** [comment]
Yeah, the same for me on my Raspberry Pi 4 whit Debian Bullseye and PHP 8.1.
I hope that the problem for me is not the Chromium Browser.
But seems that nobody fix that problem for now or have any ideas.


`Fatal error: Uncaught Spatie\Browsershot\Exceptions\CouldNotTakeBrowsershot: For some reason Chrome did not write a file at `./test.png`. Command ======= {"url":"optize.de","action":"screenshot","options":{"type":"png","path":".\/test.png","args":[],"viewport":{"width":800,"height":600},"waitUntil":"networkidle0"}} Output ====== in /home/pi/vendor/spatie/browsershot/src/Exceptions/CouldNotTakeBrowsershot.php:23 Stack trace: #0 /home/pi/vendor/spatie/browsershot/src/Browsershot.php(554): Spatie\Browsershot\Exceptions\CouldNotTakeBrowsershot::chromeOutputEmpty() #1 /var/www/html/optize/dashboard/plugins/screen.php(10): Spatie\Browsershot\Browsershot->save() #2 {main} thrown in /home/pi/vendor/spatie/browsershot/src/Exceptions/CouldNotTakeBrowsershot.php on line 23
`

**Comment by dimer47** [comment]
Hi @sajedulsakib001 @sbc-mining 

Maybe try this : https://stackoverflow.com/questions/71121581/how-to-fix-spatie-browsershot-error-somehow-chrome-did-not-write-a-file-at-scre

This work for me with [spatie/browsershot](https://github.com/spatie/browsershot) in GitHub Actions CI.
Although composer install puppetter, installing manually puppetter and chromium on the system made the generation of PDFs work.

```
npm install -g puppeteer
npm install -g chromium
```

**Comment by bemoul** [comment]
Same for me, anybody please have a solution ?

**Comment by jocket1** [comment]
same for me

**Reply by mrprotocoll** [reply]
Hi @jocket1 

were you able to find a solution?

**Comment by lance2k** [comment]
Anyone found a solution? I have the same problem

**Reply by mrprotocoll** [reply]
Hi @lance2k 

were you able to find a solution ?
